typescript/infra/src/agents/key-utils.ts

```diff
@@ -386,8 +386,8 @@ async function persistAddressesLocally(
   let relayer, kathy;
   for (const key of keys) {
     if (key.role === Role.Relayer) {
-      if (relayer)
-        throw new Error('More than one Relayer found in gcpCloudAgentKeys');
+      // if (relayer)
```

Comment:
for posterity I created https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3328 that I'll flesh out